### ISSUE TITLE
Ask for explaning

### ISSUE THREAD

**Issue by zaher88abd** [JeniaNovellusDx/KerasExtras#1]
Hi,

I read your code for Mean_IOU, I would understand how I can use it for as loss function when it returns 1 for a similar labels and less than 1 for different ones?

Do I have to put 1 - Mean_IOU(..) or what?

**Comment by Golbstein** [comment]
Yes, but generally you better use cross-entropy loss for segmentation and IOU for monitoring the performance of your model. (like "accuracy" in classification)
